docs/operations-guide/start.md

```diff
@@ -35,6 +38,25 @@ Currently in beta.  We've run Metabase on Heroku and it works just fine, but it'
 #### [Running on Cloud66](running-metabase-on-cloud66.md)
 Community support only at this time, but we have reports of Metabase instances running on Cloud66!
 
+# Upgrading Metabase
```

Comment:
This makes sense but it might be a bit easier for people to find their platform of choice by explicitly calling out Docker, the Jar, etc with a heading similar to the other platforms so they all seem like they're on an equal footing. I initially expected this section to be general background rather than specific instructions.